h2o-core/src/main/java/water/AutoBuffer.java

```diff
@@ -1008,17 +1013,70 @@ AutoBuffer putUdp(UDP.udp type, int senderPort){
    *
    * @param type type of the UDP datagram
    */
-  AutoBuffer putUdp (UDP.udp type) {
-    return putUdp(type, H2O.H2O_PORT); // Outgoing port is always the sender's (me) port
+  AutoBuffer putUdp(UDP.udp type) {
+    return putUdp(type, H2O.H2O_PORT);
+  }
+
+  /**
+   * Select last 15 bytes from the jvm boot start time and return it as char
+   */
+  public static char createUniqueId(long jvmStartTime){
```

Comment:
should this be called a timestamp? I think we should be a little bit more open about what this value is because it does depend on time and is not guaranteed to be unique if the system time is changed

it is obviously unique enough for our purposes

Reply:
@michalkurka good point, will rename!

Reply:
Implemented